Lib/fontTools/feaLib/ast.py

```diff
@@ -353,11 +382,11 @@ def asFea(self, indent=""):
 
 
 class GlyphClassDefinition(Statement):
-    """Example: @UPPERCASE = [A-Z];"""
+    """Example: ``@UPPERCASE = [A-Z];``."""
     def __init__(self, name, glyphs, location=None):
         Statement.__init__(self, location)
-        self.name = name
-        self.glyphs = glyphs
+        self.name = name #: class name as a string, without initial ``@``
+        self.glyphs = glyphs #: a :class:`GlyphClass` object
 
     def glyphSet(self):
         return tuple(self.glyphs.glyphSet())
```

Comment:
This should be documented.

Reply:
`#:` is the way you do Sphinx documentation for attributes.

Reply:
But there is no `#:` here?

Reply:
Oh, sorry! - you're talking about the `glyphSet` method? (I was just concentrating on the added lines.) OK, will fix.